Rakefile

```diff
@@ -37,12 +37,27 @@ RSpec::Core::RakeTask.new(:acceptance) do |t|
   t.pattern = 'spec/acceptance'
 end
 
-desc 'Run tests metadata_lint, release_checks'
+desc 'Run tests release_checks'
 task test: [
-  :metadata_lint,
```

Comment:
is metadata_lint called from somewhere else?

Reply:
this should be included in the release_checks task

Reply:
`validate` task calls `metadata_lint`: https://github.com/puppetlabs/puppetlabs_spec_helper/blob/db4aa99aade1b78988ab5b72078b92311bf07141/lib/puppetlabs_spec_helper/rake_tasks.rb#L211

`validate` is part of the `release_checks` task: https://github.com/puppetlabs/puppetlabs_spec_helper/blob/db4aa99aade1b78988ab5b72078b92311bf07141/lib/puppetlabs_spec_helper/rake_tasks.rb#L274